risc0/zkvm/src/host/recursion/prove/zkr.rs

```diff
@@ -45,7 +45,7 @@ pub fn test_recursion_circuit(hashfn: &str) -> Result<(Program, Digest)> {
 }
 
 pub fn lift(po2: usize, hashfn: &str) -> Result<(Program, Digest)> {
-    if (MIN_CYCLES_PO2..MAX_CYCLES_PO2).contains(&po2) {
+    if (MIN_CYCLES_PO2..=MAX_CYCLES_PO2).contains(&po2) {
```

Comment:
FYI this seems like a sorta unrelated bug, it would reject 24 for no good reason

Reply:
I don't think we have po2=24 lift predicates generated